src/test/rustdoc/issue-20727-4.rs

```diff
@@ -17,7 +17,7 @@ pub trait Index<Idx: ?Sized> {
 // @has issue_20727_4/trait.IndexMut.html
 pub trait IndexMut<Idx: ?Sized>: Index<Idx> {
     // @has - '//*[@class="rust trait"]' \
-    //        'trait IndexMut<Idx: ?Sized>: Index<Idx> {'
+    //        'trait IndexMut<Idx>: Index<Idx> where Idx: ?Sized, {'
```

Comment:
This is unfortunate, I understand that it means the same but the source not being anymore rendered the same way as it was written isn't great. If I wasn't aware of this change I would considered this a being a serious regression even-trough having the "where" clause means the same thing.

Reply:
I opened a discussion about it on [zulip](https://rust-lang.zulipchat.com/#narrow/stream/266220-rustdoc/topic/converting.20bounds.20into.20where.20bounds.3F).